models/Staticroute.php

```diff
@@ -696,6 +701,35 @@ public function match($path, $params = [])
         return [];
     }
 
+
+    /**
+     * @return array
+     */
+    public function getMethods()
+    {
+        if ($this->methods && !is_array($this->methods)) {
+            $this->methods = explode(',', $this->methods);
+        }
+
+        return $this->methods;
+    }
+
+    /**
+     * @param string|array $methods
+     *
+     * @return $this
+     */
+    public function setMethods($methods)
+    {
+        if (!is_array($methods)) {
+            $methods = strlen($methods) ? explode(',', $methods) : [];
```

Comment:
Maybe we should `trim()` the values as well.

Reply:
guess better to trim each item?

Reply:
yep, that's actually what i meant 😊

Reply:
done